requirements/base.in

```diff
@@ -9,6 +9,7 @@ dask
 graphviz
 plopp
 pythreejs
-sciline>=24.1.1
-scipp>=23.11.1
+sciline>=23.9.1
+scipp>=23.8.0
```

Comment:
Hmm why did the version of Scipp go back to 23.8...
I think we bumped to 23.11.1 because of the memory leak. But maybe we want to relax things as much as possible? It's not like 23.8 didn't work or gave wrong results, it just had a memory leak.

Reply:
:scream: :confused: good catch, why did `pip-compile-multi` do this?

Reply:
Probably because it was wrongly changed in `base.in` instead of `pyproject.toml`?

Reply:
I'll merge this, since this is unrelated.